routes/category.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const router = express.Router();
+const categoryController = require("../controllers/category");
+const   Validate  = require("../controllers/validator")
+const {upload} = require('../helpers/helper');
+router.post( '/', upload.array('categoryImages'), Validate.validate('insertCategory'), 
+    categoryController.insertCategory,
+  )
+router.put("/:id", categoryController.update);
+router.get("/",categoryController.getAll);
+router.get("/count",categoryController.countAllCategories);

```

Comment:
we don't want separate route for count, it should be in all category api

Reply:
removed